ghost/core/core/server/services/collections/BookshelfCollectionsRepository.js

```diff
@@ -105,6 +108,14 @@ module.exports = class BookshelfCollectionsRepository {
             await this.#model.destroy({id: collection.id});
```

Comment:
We should put the destroy behind a transaction too 🤔

Reply:
Good shout